packages/fxa-payments-server/src/index.scss

```diff
@@ -1,4 +1,9 @@
-@import '../../fxa-content-server/app/styles/main.scss';
+@import '../../fxa-content-server/app/styles/main';
+
+body {
+  overflow-x: hidden;
+  width: 100%;
+}
```

Comment:
When you have to use `body { overflow-x: hidden;`, it's usually a band-aid for another element causing problems.

If it's hard to tell what's causing the issue, you can temporarily throw a `* { border: 1px solid red; }` on there to see border around anything poking out.

Reply:
i was rooting around for this, but it was really hard to identify the thing causing the overflow...would you be cool with me adding a //HACK comment in CSS?

Reply:
I found the culprit! Add this:

```
#main-content {
  hr {
   ...
    @include respond-to('small') {
      margin: 0 -20px; // match padding of #main-content
    }
  }
}
```

Looks like we got around padding with a negative margin on `hr` in desktop but we didn't replicate the padding for mobile. I'm using `small` here instead of your new `simpleSmall` because the padding on `#main-content` responds to that breakpoint.

Reply:
so rad!